main.py

```diff
@@ -15,8 +15,13 @@
 from PIL import ImageTk, Image
 import eyed3
 
+    
 
 def tkinter_window(location, audiofile):
+    inputs={}
+    def get_entry():
```

Comment:
you have defined a function inside a function. Can you define it outside, it seems a bit funky

Reply:
Ill try to make it prettier but ultimately there needs to be a nested function for the callback